### PR TITLE
feat(eventsub): expose stream ID in EventSubStreamOnlineEvent

### PR DESCRIPTION
**Pull request by StimulCross**
Type: Feature
Fixes: #418

Adds `streamId` getter to access the stream ID.

### CODE REVIEW

File: packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts [hunk 1]
```diff
@@ -65,6 +65,13 @@ export class EventSubStreamOnlineEvent extends DataObject<EventSubStreamOnlineEv
 		return (await this._client.streams.getStreamByUserId(this[rawDataSymbol].broadcaster_user_id))!;
 	}
 
+	/**
+	 * The ID of the stream going live.
+	 */
+	get streamId(): string {
```
Comment (d-fischer) [inline packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts:71]: Should just be named `id` IMO. The event already refers to a stream, so no need to qualify.

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts (hunk 1 above)
Comment (StimulCross) [reply packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts:71]: Well, I can rename it. But I chose this name to be consistent with `streamType` getter.

```ts
get streamType(): EventSubStreamOnlineEventStreamType {
    return this[rawDataSymbol].type;
}
```

Is it ok?

Comment (d-fischer) [reply packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts:71]: Fair enough, I will probably have to clean up a bunch of naming inconsistencies when I can. As for this one, please just rename it to `id`.

Comment (StimulCross) [reply packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts]: Done 👀

**Review by d-fischer — APPROVED** [review]

**Review by d-fischer — APPROVED** [review]

**Pull request merged**